src/components/form.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Form = () => (
+  <>
+    <h2>ADD NEW BOOK</h2>
+    <form>
+      <input type="text" />
+      <select name="categorie" id="categorie">
```

Comment:
[ Optional Change ]

- [ ] It would be nice if you could add a placeholder for the title of the book in the input field so one knows they can enter a book title there. 👍